docs/learn-accounts.md

```diff
@@ -81,3 +81,43 @@ Because of this unreliability of indices, they **may not be present** in Polkado
 ## Identities
 
 The _Identities_ pallet built into Kusama allows users to attach on-chain metadata to their accounts. This metadata can be verified by independent registrars to provide trustworthiness. To learn more about how to set or release an identity, how to define sub-accounts, or how to become a registrar, please read [this guide](learn-identity).
+
+## Multi-signature Accounts
+
+It is possible to create a multi-signature account in Substrate-based chains. A multi-signature account is composed of one or more addresses and a threshold. The threshold defines how many signatories (participating addresses) need to agree on the submission of an extrinsic in order for the call to be successful. 
+
+For example, Alice, Bob, and Charlie set up a multi-sig with a threshold of 2. This means Alice and Bob can execute any call even if Charlie disagrees with it. Likewise, Charlie and Bob can execute any call without Alice. A threshold is typically a number smaller than the total number of members but can also be equal to it which means they all have to be in agreement.
+
+Multi-signature accounts have several uses:
+
+- securing your own stash: use additional signatories as a 2FA mechanism to secure your funds. One signer can be on one computer, another can be on another, or in cold storage. This slows down your interactions with the chain, but is orders of magnitude more secure.
+- board-decisions: legal entities such as businesses and foundations use multi-sigs to collectively govern over the entity's treasury.
+- group participation in governance: a multi-sig account can do everything a regular account can. A multi-sig account could be a council member in Kusama's governance, where a subset of community members could vote as one entity.
```

Comment:
```suggestion
- group participation in governance: a multi-sig account can do everything a regular account can. A multi-sig account could be a council member in Kusama's governance, where a set of community members could vote as one entity.
```
I know that "subset" is _technically_ correct here, but I think set makes more sense on an informal basis